src/main/java/core/basesyntax/model/ma/Coach.java

```diff
@@ -8,6 +13,15 @@
     private int experience;
     private Track track;
```

Comment:
better to put @Enumerated(EnumType.STRING) annotation as well on the field where you use it